extension/src/nodes/blackboard.cpp

```diff
@@ -0,0 +1,20 @@
+#include "blackboard.h"
+
+#include <godot_cpp/core/class_db.hpp>
+
+using namespace godot;
+
+BeehaveBlackboard::BeehaveBlackboard()
+{
+    count = 0;
+}
+
+BeehaveBlackboard::~BeehaveBlackboard()
+{
+    
+}
+
+void BeehaveBlackboard::_bind_methods()
+{
+
+}
```

Comment:
nit: missing new line at the end of file. This is usually not a big deal, but might cause some [inconsistent or unexpected behavior in some cases](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file). Plus, github would always warn you about it, as it does here.

P.S. Most popular IDEs these days has an option to auto-add end-of-file-new-line on Save of file close - so you never would need to think about this again.